src/cpp/desktop/DesktopOptions.cpp

```diff
@@ -347,15 +347,15 @@ void Options::setClipboardMonitoring(bool monitoring)
    settings_.setValue(QString::fromUtf8("clipboard.monitoring"), monitoring);
 }
 
-bool Options::ignoreGpuBlacklist() const
+bool Options::ignoreGpuExclusionList() const
 {
-   QVariant ignore = settings_.value(QStringLiteral("general.ignoreGpuBlacklist"), false);
+   QVariant ignore = settings_.value(QStringLiteral("general.ignoreGpuExclusionList"), false);
```

Comment:
Does this imply that the user's old desktop options here would be lost on upgrade? (That's probably fine but wanted to double-check it's a conscious decision)

Reply:
Yep, that's what I meant to do. Losing this option just means that we will start using software rendering (the safer option) so a small handful of people will need to re-enable the option to use acceleration again.